commit fff64d9fb3817fee88ce211bd4fef94558d877be
Author: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
Date:   Sun Jun 10 19:45:32 2018 +0200

    EDAC, altera: Fix an error handling path in altr_s10_sdram_probe()
    
    [ Upstream commit 9d72fe1ce81bc757ecb6d57b58e5fd95b9ad1b26 ]
    
    If regmap_write() fails, we should release some resources as done in all
    the other error handling paths of the function.
    
    Signed-off-by: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
    Reviewed-by: Thor Thayer <thor.thayer@linux.intel.com>
    Cc: linux-edac <linux-edac@vger.kernel.org>
    Link: http://lkml.kernel.org/r/20180610174532.22071-1-christophe.jaillet@wanadoo.fr
    Fixes: e9918d7fafae ("EDAC, altera: Handle SDRAM Uncorrectable Errors on Stratix10")
    Signed-off-by: Borislav Petkov <bp@suse.de>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/edac/altera_edac.c b/drivers/edac/altera_edac.c
index d0d5c4dbe097..5762c3c383f2 100644
--- a/drivers/edac/altera_edac.c
+++ b/drivers/edac/altera_edac.c
@@ -730,7 +730,8 @@ static int altr_s10_sdram_probe(struct platform_device *pdev)
 			 S10_DDR0_IRQ_MASK)) {
 		edac_printk(KERN_ERR, EDAC_MC,
 			    "Error clearing SDRAM ECC count\n");
-		return -ENODEV;
+		ret = -ENODEV;
+		goto err2;
 	}
 
 	if (regmap_update_bits(drvdata->mc_vbase, priv->ecc_irq_en_offset,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1207_fff64d9fb3817fee88ce211bd4fef94558d877be.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1207_fff64d9fb3817fee88ce211bd4fef94558d877be.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
